rise-financial-es6.js

```diff
@@ -214,6 +214,18 @@
       financial.params = params;
     }
 
+    _handleNoNetwork() {
+      this.$.data.getItem( this._getDataCacheKey(), ( cachedData ) => {
+        if ( cachedData ) {
+          this.fire( "rise-financial-response", cachedData );
+        } else {
+          this.fire( "rise-financial-no-network" );
```

Comment:
@stulees `rise-financial-no-network` seems a bit too specific. `rise-financial-no-data` or similar is more generic and would potentially allow the event to be reused in other future scenarios.